source/synthDrivers/sapi4.py

```diff
@@ -183,15 +194,19 @@ def speak(self,speechSequence):
 		)
 
 	def cancel(self):
-		self._ttsCentral.AudioReset()
-		self.lastIndex=None
+		try:
+			self._ttsCentral.AudioReset()
+		except COMError:
+			log.error("Error cancelling speech", exc_info=True)

```

Comment:
is there a reason why this logs at a different level to the `pause` changes

Reply:
Errors for pause just passed, I didn't want to introduce logging at the error level for this, though it made sense to me to at least something. Cancel, on the other hand, didn't catch any errors, so it made sense to log them at the error level.